tests/test_parser.py

```diff
@@ -694,3 +694,41 @@ def test_expand_concatenated_variables(self, dfparser):
             LABEL component="$NAME$VER"
         """)
         assert dfparser.labels['component'] == 'name1'
+
+    def test_multiline(self, dfparser):
+        dfparser.content = dedent("""
+            LABEL a=b c d e \
+                  b="parsing is so hard" \
+                  c=y e s
+        """)
+        assert dfparser.labels == {
+            "a": "b c d e",
+            "b": "parsing is so hard",
+            "c": "y e s"
+        }
+
+    def test_multiline2(self, dfparser):
+        dfparser.content = dedent("""
+            LABEL a="b c" \
+                  b=parsing is so hard \
+                  c="y e s"
```

Comment:
This test case is not valid because it mixes and matches the two forms of labels within a statement.
```
Sending build context to Docker daemon 2.048 kB
Error response from daemon: Syntax error - can't find = in "is". Must be of the form: name=value
```